addon/types/index.ts

```diff
@@ -70,8 +70,8 @@ export interface ChangesetDef {
   _runningValidations: RunningValidations,
   _bareChanges: { [s: string]: any },
 
-  changes: any, // { key: string; value: any; }[], //ComputedProperty<object[], object[]>,
-  errors: { key: string; value: any; validation: ValidationErr | ValidationErr[] }[], //ComputedProperty<object[], object[]>,
```

Comment:
@snewcomer Any specific reason for this change? It was super helpful to have the actual shape of the error here.  I would suggest making a type (or an interface) specifically for the error. 

Here is an example of a function that is written in TS which was working fine before this change:

```ts
  get errors(): string[] {
    const fieldErrors = this.args.changeset.errors.filter(error => {
      return error.key === this.fullFieldName;
    });

    return fieldErrors.reduce(
      (errors: string[], error: ValidationError): string[] => {
        return [...errors, ...error.validation];
      },
      []
    );
  }
```

Now typescript is not happy, with the following error: 

```
Property 'key' does not exist on type 'object'.

25       return error.key === this.fullFieldName;
```

Probably the same would apply for `changes` above.

Reply:
This should be an easy fix...